applications/tari_console_wallet/src/grpc/wallet_grpc_server.rs

```diff
@@ -196,6 +265,7 @@ fn convert_wallet_transaction_into_transaction_info(
             excess_sig: Default::default(),
             timestamp: Some(naive_datetime_to_timestamp(tx.timestamp)),
             message: tx.message,
+            valid: true,
```

Comment:
`valid` as a term is a bit vague. Under what conditions is `valid == false`?

Reply:
This exactly mimics the behaviour of 
   `impl From<OutboundTransaction> for CompletedTransaction {`
 and 
   `impl From<InboundTransaction> for CompletedTransaction {`.


 The meaning of `valid` in `InboundTransaction` and `OutboundTransaction` is actually not defined and therefore always `true`; it only gets to mean something if the transaction progressed from `PendingInbound` or `PendingOutbound` to `Completed`.